How-we-use-pull-requests.md

```diff
@@ -0,0 +1,68 @@
+## Table of contents
+
+* [PR lifecycle](#pr-lifecycle)
+* [Draft pull requests](#draft-pull-requests)
+* [Labeling issues and pull requests](#labeling-issues-and-pull-requests)
+* [Juggling multiple pull requests](#juggling-multiple-pull-requests)
+
+## PR lifecycle
```

Comment:
Should we call it a Pull request lifecycle? (Considering all other headings have "pull request")

Reply:
done